### ISSUE TITLE
Warning message about "SensorFusion: Mag sample is too old" flooding logcat

### ISSUE THREAD

**Issue by isometriq** [googlevr/gvr-android-sdk#530]
As soon as a create a GvrView instance, I receive a ton of warning messages (every milliseconds or so) and it is flooding the logs. It is more annoying as I don't know how to filter them out. Also, I'm not sure if it could be an issue. Here is a small sample of the warnings I get:
```
01-22 12:49:19.613 18459-18578/com.testing.gvr W/native: orientation_filter.cc:400 SensorFusion: Mag sample is too old: 0.00616455
01-22 12:49:19.633 18459-18578/com.testing.gvr W/native: orientation_filter.cc:400 SensorFusion: Mag sample is too old: 0.00558472
01-22 12:49:19.653 18459-18578/com.testing.gvr W/native: orientation_filter.cc:400 SensorFusion: Mag sample is too old: 0.00546265
01-22 12:49:19.672 18459-18578/com.testing.gvr W/native: orientation_filter.cc:400 SensorFusion: Mag sample is too old: 0.00518799
01-22 12:49:19.692 18459-18578/com.testing.gvr W/native: orientation_filter.cc:400 SensorFusion: Mag sample is too old: 0.00482178
```
Phone reporting the warnings is a Samsung S8. I don't get these in the emulator. I will test with other physical devices when I can.

**Comment by isometriq** [comment]
Tested on another phone (LG G6) and I did not get the messages. Will try other ones as the occasion arises.

**Comment by isometriq** [comment]
Tested other ones: Samsung S6 and Samsung S7.
I did not get the message also on these.

Only on the Samsung S7, these 2 warnings appearing when shutting down:
```
01-22 14:44:50.096 18029-18745/com.testing.gvr W/native: sensor_fusion_mahony.cc:189 Invalid timestamps detected.  Time step between successive gyroscope sensor samples exceeded threshold. Resetting orientation filter and bias.
01-22 14:45:05.576 18029-18777/com.testing.gvr W/native: sensor_fusion_mahony.cc:189 Invalid timestamps detected.  Time step between successive gyroscope sensor samples exceeded threshold. Resetting orientation filter and bias.
```

**Comment by jdduke** [comment]
Hi @isometriq, thanks for the report, we're investigating this internally.

If you get a chance to capture a bug report of the issue on your S8, that would be extremely helpful. In particular, it would help to know the exact device name/model, what version of Android you're running, and what version of Google VR Services you have installed. Thanks.

**Comment by isometriq** [comment]
Can you point in the right direction to create such a report ?

I can provide the device info, but if the report includes it, I would rather use that.

**Comment by jdduke** [comment]
If you have the Android SDK, or access to adb, you can use:

  adb bugreport s8_gvr_bug.zip

There are additional [instructions here](https://developer.android.com/studio/debug/bug-report.html) if you don't have access to adb. I'll pm you my @google.com email address to which you can send the report. Thanks.

**Comment by jdduke** [comment]
The log spam will be removed in the next SDK release, expected next week.

**Comment by jdduke** [comment]
This should no longer be an issue with the latest release.

**Comment by ptran-gpfw** [comment]
On Samsung Note 8 (US, SM-N950U1) I still observe this with release `1.130.0`.

> 03-21 17:02:20.243 12067-15854/com.gopro.smarty W/native: orientation_filter.cc:399 SensorFusion: Mag sample is too old: 0.00494385

**Comment by jdduke** [comment]
You're absolutely right, looks like the fix just missed the 1.30 release. It will be in the next release, 2 weeks from now. I'll reopen until that is pushed.

**Comment by jdduke** [comment]
Fixed in the 1.40.0 release.
